Documentation/contributing.rst

```diff
@@ -785,33 +785,6 @@ Second stage (unstable versions)
     - 1.10 beta
     - 1.11 alpha
 
-CI / Testing environment
```

Comment:
Let's keep this section. The instances are accessible for any core members.

Reply:
Sorry for the delay on getting this done, fixed.